### ISSUE TITLE
Question for the data.

### ISSUE THREAD

**Issue by sev777** [nlpersECJTU/LDSGM#2]
Hi,   

(1) The data size in this paper  is "the training set with 12,775 instances (Section 2-20), the validation set with 1,183 instances (Section 0-1), and the test set with 1,046 instances (Section 21-22)"， Is this before or after processing?

  *Processing means: "Further, there exist 16 second-level labels, five of which with few training instances and no validation and test instance are removed. Therefore, we conduct an 11-way classification on the second-level labels. " in the paper.

(2) And in paper "On the Importance of Word and Sentence Representation Learning in Implicit Discourse Relation Classification", the data size is  Train/dev/test: 12362, 1183, 1046.     Which one is right and If there any code to process the dataset.

Thank you!
